File: src/main/scala/org/openapitools/server/model/UpdateOfferMappingsResponse.scala
package org.openapitools.server.model


/**
 * Описывает проблемы, возникшие при сохранении товара.
 *
 * @param status  for example: ''null''
 * @param results Ошибки и предупреждения, которые появились при обработке списка характеристик. Каждый элемент списка соответствует одному товару.  Если ошибок и предупреждений нет, поле не передается.  for example: ''null''
*/
final case class UpdateOfferMappingsResponse (
  status: Option[ApiResponseStatusType] = None,
  results: Option[Seq[UpdateOfferMappingResultDTO]] = None
)

File: ympa_fsharp_functions_server/src/model/UpdateOfferMappingsResponse.fs
namespace ympa_fsharp_functions_server.Model

open System
open System.Collections.Generic
open Newtonsoft.Json
open ympa_fsharp_functions_server.Model.ApiResponseStatusType
open ympa_fsharp_functions_server.Model.UpdateOfferMappingResultDTO

module UpdateOfferMappingsResponse =

  //#region UpdateOfferMappingsResponse

  [<CLIMutable>]
  type UpdateOfferMappingsResponse = {
    [<JsonProperty(PropertyName = "status")>]
    Status : ApiResponseStatusType;
    [<JsonProperty(PropertyName = "results")>]
    Results : UpdateOfferMappingResultDTO[];
  }

  //#endregion
